app/models/hearing_day.rb

```diff
@@ -186,6 +238,24 @@ def only_changed(possibles_hash)
     changed_hash
   end
 
+  # Creates a datetime with timezone from these parts
+  # - Time, a string like "08:30"
+  # - Timezone, a string like "America/Los_Angeles"
+  # - Date, a ruby Date
+  # :reek:UtilityFunction
+  def combine_time_and_date(time, timezone, date)
```

Comment:
This would reduce the number of arguments passed to this method in method `begins_at`
```suggestion
  def combine_time_and_date(time, timezone = "America/New_York", date = scheduled_for)
```

Reply:
Wanted to note that I ended up removing these defaults because I think the intention is much clearer without them.

Without defaults `combine_time_and_date("09:00", "America/New_York", scheduled_for)`
- Reads as 'oh, this function takes a time, zone, and date and combines them somehow'

With defaults `combine_time_and_date("09:00")`
- I immediately want to know why this function is named the way it is, also what is the date it's combining?